app/script/util/util.coffee

```diff
@@ -94,7 +94,10 @@ z.util.get_url_parameter = (name) ->
   for param in params
     value = param.split '='
     if value[0] is name
-      return if value[1] then unescape value[1] else 'true'
+      if value[1]
+        value = unescape value[1]
+        return if value is 'false' then false else value
+      return true
   return null
```

Comment:
I know this grew over the last weeks. Now all needed (and possible cases) are inclided. Unfortunately we need all of them as "false" obviuosly isnt false and that check failed numourous times. Any suggestion for cleaner code appreciated.